scripts/verify_tla_bridge_coverage.py

```
@@ -0,0 +1,204 @@
+    """Parse INVARIANTS and PROPERTIES from a .cfg file."""
+    with open(cfg_path) as f:
+        lines = f.readlines()
+    invariants: list[str] = []
+    properties: list[str] = []
+    section = None
+    for raw in lines:
+        line = raw.strip()
+        if line == "INVARIANTS":
+            section = "inv"
+            continue
+        elif line == "PROPERTIES":
```

Ruff (0.15.10)

[warning] Unnecessary `elif` after `continue` statement

Remove unnecessary `elif`

(RET507)